### DISCUSSION TITLE
Potential issue with amount of goroutines with 0.14rc3

### DISCUSSION THREAD

**Discussion by Dikay900** [lightningnetwork/lnd#5963] [Troubleshooting]
I am currently testing the latest lnd 0.14 rc3 and I did find something while monitoring the node with lndmon:
![tmp](https://user-images.githubusercontent.com/448549/141294540-6235618c-6409-4c04-a169-4725b2bbb14f.png)

Since upgrading to the RC the num of goroutines is increasing. With the last release of lnd0.13.3 everyhting was stable as you can see. For now everything is running smoothly and no further impact is visible but still I wanted to raise awareness if this is a problem or just as intended?

Edit: I digged a bit deeper and it seems the memory usage of the lnd process is increasing slightly together with the num of goroutines:
![tmp2](https://user-images.githubusercontent.com/448549/141295871-e4a9dbea-51c2-4379-b230-6e082df5dd1f.png)

**Comment by Crypt-iQ** [comment]
This seems like a leak, if you have any logs it would be helpful to make an issue on the repo
